.github/workflows/test.yml

```diff
@@ -40,3 +40,40 @@ jobs:
           restore-keys: ${{ runner.os }}-jdk${{ matrix.java }}-
       - name: Test
         run: mvn test
+
+  test_graalvm:
+    name: test ${{ matrix.os }} ${{ matrix.java }} GraalVM native-image
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
As the GitHub Action capacity is not infinite, I'd like to limit the scope of GraalVM only to the lastest JDK (java11 or java16) and Linux.

Reply:
Agreed, I use graalvm only with jdk11 for clojure-lsp test CI as well

Reply:
@xerial what do you mean it's not infinite? There's no credit or anything for OSS projects AFAIK.

Reply:
@gotson there is no credit, but there are hardware limitation when you spawn a lot of heavy processing actions GitHub only run new jobs when others finish

Reply:
I think they run in order, so the classic tests run first, and the graalvm ones run after. I'm not sure we need feedback that fast to be honest?

There are differences between the different platforms though. I agree on java 8, we can drop it and keep only 11 for now.

Ultimately it's up to @xerial, I am happy to change it anyway you prefer!